<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <layout class="QGridLayout" name="mainLayout">
      <item row="0" column="1">
       <widget class="VideoPlayer" name="videoPlayer"/>
      </item>
      <item row="0" column="0">
       <widget class="WaveformView" name="waveform" native="true"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QTableView" name="subtitleList">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QFrame" name="buttons">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionOpenRecent"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionProject_Properties"/>
    <addaction name="actionOpen_Project_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionZip_Subtitle"/>
    <addaction name="actionTranslate"/>
    <addaction name="actionExport_to_WAV"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Presets"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_next"/>
    <addaction name="actionGoto"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_file"/>
    <addaction name="actionText_pipe"/>
    <addaction name="actionShow_Text_VO_Other_diffs"/>
    <addaction name="separator"/>
    <addaction name="actionError_checking"/>
    <addaction name="actionSpell_check"/>
    <addaction name="actionDelay"/>
    <addaction name="actionQuick_Smart_Delay"/>
    <addaction name="actionSmart_desynch_tool"/>
    <addaction name="actionFps_adaptation_tool"/>
    <addaction name="separator"/>
    <addaction name="actionStyles"/>
    <addaction name="actionWeb_server"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Selection"/>
    <addaction name="actionZoom_All"/>
    <addaction name="actionZoom_Vertical"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Hide_Reference_VO"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Hide_Video"/>
    <addaction name="actionSet_Video_Resolution"/>
    <addaction name="actionShow_Hide_scene_change"/>
    <addaction name="actionShow_Hide_subtitles"/>
    <addaction name="actionDetach_video_window"/>
    <addaction name="actionShow_frame_at_cursor"/>
    <addaction name="actionShow_start_frame"/>
    <addaction name="actionShow_stop_frame"/>
    <addaction name="separator"/>
    <addaction name="actionShow_silent_zones"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Hide_Logs"/>
   </widget>
   <widget class="QMenu" name="menuPlayback_Tools">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
    <addaction name="actionLoop"/>
    <addaction name="actionNext_subtitle"/>
    <addaction name="actionPrevious_subtitle"/>
    <addaction name="actionLoop_selection_start"/>
    <addaction name="actionLoop_selection_end"/>
    <addaction name="actionPlay_selection_start"/>
    <addaction name="actionPlay_selection_end"/>
    <addaction name="actionPlay_to_end"/>
    <addaction name="separator"/>
    <addaction name="actionPlayback_Rate"/>
    <addaction name="actionWave_form_correction"/>
   </widget>
   <widget class="QMenu" name="menuJS_Tools">
    <property name="title">
     <string>JS Tools</string>
    </property>
    <addaction name="actionSplit_line"/>
    <addaction name="actionFix_typos"/>
    <addaction name="actionAdd_sharp"/>
    <addaction name="actionRemove_Tags"/>
    <addaction name="actionRemove_Apostrophes"/>
    <addaction name="actionRemove_All"/>
    <addaction name="actionQuick_stats"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionCheck_for_updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPlayback_Tools"/>
   <addaction name="menuJS_Tools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionOpenRecent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionProject_Properties">
   <property name="text">
    <string>Project Properties</string>
   </property>
  </action>
  <action name="actionOpen_Project_Folder">
   <property name="text">
    <string>Open Project Folder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionZip_Subtitle">
   <property name="text">
    <string>Zip Subtitle</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="text">
    <string>Translate</string>
   </property>
  </action>
  <action name="actionExport_to_WAV">
   <property name="text">
    <string>Export to WAV</string>
   </property>
  </action>
  <action name="actionLoad_Presets">
   <property name="text">
    <string>Load Presets</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_next">
   <property name="text">
    <string>Find next</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>Goto</string>
   </property>
  </action>
  <action name="actionInsert_file">
   <property name="text">
    <string>Insert file</string>
   </property>
  </action>
  <action name="actionText_pipe">
   <property name="text">
    <string>Text pipe</string>
   </property>
  </action>
  <action name="actionShow_Text_VO_Other_diffs">
   <property name="text">
    <string>Show Text &lt;-&gt;VO/Other diffs</string>
   </property>
  </action>
  <action name="actionError_checking">
   <property name="text">
    <string>Error checking</string>
   </property>
  </action>
  <action name="actionSpell_check">
   <property name="text">
    <string>Spell check</string>
   </property>
  </action>
  <action name="actionDelay">
   <property name="text">
    <string>Delay</string>
   </property>
  </action>
  <action name="actionQuick_Smart_Delay">
   <property name="text">
    <string>Quick &amp;&amp; Smart Delay</string>
   </property>
  </action>
  <action name="actionSmart_desynch_tool">
   <property name="text">
    <string>Smart desynch tool</string>
   </property>
  </action>
  <action name="actionFps_adaptation_tool">
   <property name="text">
    <string>Fps adaptation tool</string>
   </property>
  </action>
  <action name="actionStyles">
   <property name="text">
    <string>Styles</string>
   </property>
  </action>
  <action name="actionWeb_server">
   <property name="text">
    <string>Web server</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_Selection">
   <property name="text">
    <string>Zoom Selection</string>
   </property>
  </action>
  <action name="actionZoom_All">
   <property name="text">
    <string>Zoom All</string>
   </property>
  </action>
  <action name="actionZoom_Vertical">
   <property name="text">
    <string>Zoom Vertical</string>
   </property>
  </action>
  <action name="actionShow_Hide_Reference_VO">
   <property name="text">
    <string>Show/Hide Reference VO</string>
   </property>
  </action>
  <action name="actionShow_Hide_Video">
   <property name="text">
    <string>Show/Hide Video</string>
   </property>
  </action>
  <action name="actionSet_Video_Resolution">
   <property name="text">
    <string>Set Video Resolution</string>
   </property>
  </action>
  <action name="actionShow_Hide_scene_change">
   <property name="text">
    <string>Show/Hide scene change</string>
   </property>
  </action>
  <action name="actionShow_Hide_subtitles">
   <property name="text">
    <string>Show/Hide subtitles</string>
   </property>
  </action>
  <action name="actionDetach_video_window">
   <property name="text">
    <string>Detach video window</string>
   </property>
  </action>
  <action name="actionShow_frame_at_cursor">
   <property name="text">
    <string>Show frame at cursor</string>
   </property>
  </action>
  <action name="actionShow_start_frame">
   <property name="text">
    <string>Show start frame</string>
   </property>
  </action>
  <action name="actionShow_stop_frame">
   <property name="text">
    <string>Show stop frame</string>
   </property>
  </action>
  <action name="actionShow_silent_zones">
   <property name="text">
    <string>Show silent zones</string>
   </property>
  </action>
  <action name="actionShow_Hide_Logs">
   <property name="text">
    <string>Show/Hide Logs</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionNext_subtitle">
   <property name="text">
    <string>Next subtitle</string>
   </property>
  </action>
  <action name="actionPrevious_subtitle">
   <property name="text">
    <string>Previous subtitle</string>
   </property>
  </action>
  <action name="actionLoop_selection_start">
   <property name="text">
    <string>Loop selection start</string>
   </property>
  </action>
  <action name="actionLoop_selection_end">
   <property name="text">
    <string>Loop selection end</string>
   </property>
  </action>
  <action name="actionPlay_selection_start">
   <property name="text">
    <string>Play selection start</string>
   </property>
  </action>
  <action name="actionPlay_selection_end">
   <property name="text">
    <string>Play selection end</string>
   </property>
  </action>
  <action name="actionPlay_to_end">
   <property name="text">
    <string>Play to end</string>
   </property>
  </action>
  <action name="actionPlayback_Rate">
   <property name="text">
    <string>Playback Rate</string>
   </property>
  </action>
  <action name="actionWave_form_correction">
   <property name="text">
    <string>Wave form correction</string>
   </property>
  </action>
  <action name="actionSplit_line">
   <property name="text">
    <string>Split line</string>
   </property>
  </action>
  <action name="actionFix_typos">
   <property name="text">
    <string>Fix typos</string>
   </property>
  </action>
  <action name="actionAdd_sharp">
   <property name="text">
    <string>Add sharp</string>
   </property>
  </action>
  <action name="actionRemove_Tags">
   <property name="text">
    <string>Remove Tags</string>
   </property>
  </action>
  <action name="actionRemove_Apostrophes">
   <property name="text">
    <string>Remove Apostrophes</string>
   </property>
  </action>
  <action name="actionRemove_All">
   <property name="text">
    <string>Remove All</string>
   </property>
  </action>
  <action name="actionQuick_stats">
   <property name="text">
    <string>Quick stats</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QOpenGLWidget</extends>
   <header>videoPlayer/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>WaveformView</class>
   <extends>QWidget</extends>
   <header>waveformView/waveformview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
